test/integration/push_int_test.go

```diff
@@ -90,7 +91,7 @@ func (suite *PushIntegrationTestSuite) TestCarlisle() {
 	ts.LoginAsPersistentUser()
 
 	// convert to real project
-	cp = ts.SpawnWithOpts(e2e.WithArgs("init", namespace), e2e.WithWorkDirectory(wd))
+	cp = ts.SpawnWithOpts(e2e.WithArgs("init", namespace, "python3"), e2e.WithWorkDirectory(wd))
```

Comment:
When transformed from a headless commit, the language should be inferred.  No need to specify it.

Reply:
Yes you're correct, this is from debugging